spec/support/vcr.rb

```diff
@@ -21,9 +21,12 @@
 
   c.register_request_matcher :html_body_content do |request_one, request_two|
     def clean(string)
-      string.gsub(/\"[^\"]+\"/, "")
+      string.gsub(/""/, "\"FOO\"").gsub(/"[^"]*"/, "\"Classes\"").gsub(/></, ">BAR<").gsub(/>[^<>]*</, ">Info<")
```

Comment:
I think it'd be useful to have a comment here explaining what this line is doing.